pandas/tests/indexing/test_loc.py

```diff
@@ -1552,6 +1552,40 @@ def test_loc_getitem_str_timedeltaindex(self):
         sliced = df.loc["0 days"]
         tm.assert_series_equal(sliced, expected)
 
+    @pytest.mark.parametrize("indexer_end", [None, "2020-01-02 23:59:59.999999999"])
+    def test_loc_getitem_partial_slice_non_monotonicity(
+        self, indexer_end, frame_or_series
+    ):
+        # GH#33146
+        df = frame_or_series(
```

Comment:
can you call this `obj` instead of `df` (which is generally DataFrame-specific)

Reply:
Done. Was before parametrization probably